### PR TITLE
Parallel constant generation

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: executor/src/constant_evaluator/mod.rs [hunk 1]
```diff
@@ -37,6 +38,7 @@ fn generate_values(
 ) -> Vec<FieldElement> {
     match body {
         FunctionValueDefinition::Mapping(body) => (0..degree)
+            .into_par_iter()
```
Comment (leonardoalt) [inline executor/src/constant_evaluator/mod.rs:41]: how many cores is this gonna use?

**Comment by chriseth** [comment]
Not sure this actually provides a speedup. Probably too much communication overhead.

**Comment by leonardoalt** [comment]
really? that's kinda surprising

**Comment by chriseth** [comment]
Ok, I guess it depends on the constants. Constant generation in `global.pil` goes from 0.3 seconds to 0.12 seconds.

**Comment by leonardoalt** [comment]
also maybe with more rows we'll see a bigger gain

**Review by Schaeff — APPROVED** [review]

**Pull request merged**